apps/sms/services/js/moz_mobile_message/messaging_service.js

```diff
@@ -0,0 +1,193 @@
+/*global BridgeServiceMixin, Settings */
```

Comment:
I think MessagingService should be in its own directory -- it's not part of the moz_mobile_message ship, it's only using it.